app/views/users/_form.html.erb

```diff
@@ -36,6 +36,16 @@
     <%= f.text_field :address %>
   </div>
 
+  <div class="field">
+    <%= f.label :avatar %>
+    <%= f.file_field :avatar %>
+  </div>
+
+   <div class="field">
+    <%= f.label :attachment %>
+    <%= f.file_field :attachment %>
```

Comment:
Until we don't have attachments handler for user, I think that we can comment it out.